modules/gdscript/gdscript_parser.cpp

```diff
@@ -3582,6 +3583,31 @@ bool GDScriptParser::export_annotations(const AnnotationNode *p_annotation, Node
 			variable->export_info.hint_string = hint_prefix + ":" + variable->export_info.hint_string;
 			variable->export_info.type = Variant::ARRAY;
 		}
+
+		if (p_annotation->name == SNAME("@export_action")) {
+			variable->export_info.hint_string = "action";
+			if (!p_annotation->resolved_arguments.is_empty()) { // Icon Name
+				variable->export_info.hint_string += (String)p_annotation->resolved_arguments[0];
+			}
+			variable->export_info.usage = PROPERTY_USAGE_EDITOR;
```

Comment:
Is this really needed? No other export uses this, so I wonder.

Reply:
This was marked as resolved but I don't see what the resolution was?